toxcore/group.c

```diff
@@ -509,7 +509,7 @@ static int note_peer_active(Group_Chats *g_c, uint32_t groupnumber, uint16_t pee
 
         Group_Peer *frozen_temp = (Group_Peer *)realloc(g->frozen, sizeof(Group_Peer) * (g->numfrozen));
 
-        if (frozen_temp == nullptr && g->numfrozen > 0) {
+        if (frozen_temp == nullptr) {
```

Comment:
Are you sure about this?

Reply:
Yes. This is something robinli pointed out. We're in the else branch of an "if (g->numfrozen == 0)" (and we know from get_frozen_peers that g->numfrozen is non-negative).

Reply:
Ah yes, I see.